website/src/pages/v1/serve/deployment/docker.mdx

```diff
@@ -125,48 +125,76 @@ need to create a new Dockerfile basing the image off `ghcr.io/ardatan/mesh-serve
 For example, adding [Rate Limiting](/v1/serve/features/security/rate-limiting) to the container
 would look like this:
 
+### Install plugin
+
+If you want to only add a plugin (or some other dependencies), you can simply extend the image and
+install the modules with `npm i`:
+
 ```dockerfile filename="Dockerfile"
 FROM ghcr.io/ardatan/mesh-serve
 
 RUN npm i @graphql-mesh/plugin-rate-limit
 ```
 
-Build your image:
+### Develop additional resolvers
 
-```sh
-docker build -t mesh-serve-w-rate-limit .
+However, you may have additional resolvers and have a more sophisticated setup with some
+dependencies and source code, copying over your project's files is the way to go.
+
+Say you have the following files:
+
+```json filename="package.json"
+{
+  "name": "my-time",
+  "dependencies": {
+    "moment": "^2"
+  },
+  "devDependencies": {
+    "@graphql-mesh/serve-cli": "latest"
+  }
+}
 ```
 
-Configure to use the rate limiting plugin:
+```js filename="my-time.ts"
+import moment from 'moment'
+
+export const additionalResolvers = {
```

Comment:
This is way better <3

Reply:
I've changed the docs to showcase developing a plugin instead.

https://github.com/ardatan/graphql-mesh/pull/7330/commits/b92e7de30e6e64947adf0f598034bcf72678069b

Reply:
Added both, why not.